test/cache.ts

```diff
@@ -134,7 +134,7 @@ test('doesn\'t cache response when received HTTP error', withServer, async (t, s
 
 test('DNS cache works', withServer, async (t, _server, got) => {
 	const map = new Map();
-	await t.notThrowsAsync(got('https://example.com', {dnsCache: map}));
+	await t.notThrowsAsync(got('https://example.com', {dnsCache: map, prefixUrl: null}));
```

Comment:
Why are you passing in `prefixUrl: null` here?

Reply:
https://github.com/sindresorhus/got/pull/829#discussion_r307972807

(it doesn't matter if the string's empty or it's null)

Reply:
Then I think you should pass an empty string here too for consistency and clarity.